src/mame/drivers/midqslvr.cpp

```diff
@@ -2,25 +2,254 @@
 // copyright-holders:Angelo Salese
 /***************************************************************************
 
-    Midway Quicksilver skeleton driver
-
-    TODO:
-    - offrthnd: illegal opcode tripped just after that PIIX4 is recognized
-
-    Main CPU : Intel Celeron 333/366MHz
-    Motherboard : Intel SE440BX-2
-    RAM : 64MB PC100-222-620 non-ecc
-    Sound: Integrated YMF740G
-    Networking: SMC EZ Card 10 / SMC1208T (probably 10ec:8029 1113:1208)
-    Graphics Chips : Quantum Obsidian 3DFX
-    Storage : Hard Drive
-
-    Chipsets (440BX AGPset):
-    - 82371EB PCI-ISA bridge
-    - 82371EB Power Management Controller
-    - 82371AB/EB Universal Host Controller (USB UHCI)
-    - 82371AB/EB PCI Bus Master IDE Controller
-
+Midway Quicksilver II/Graphite skeleton driver
+	
+Hardware configurations:
+
+Hydro Thunder:   Quicksilver II system and Diego I/O board
+Offroad Thunder: Quicksilver II system and Magicbus I/O board	
+Arctic Thunder:  Graphite system and Substitute I/O board
+	
+All of the games communicate with their I/O boards serially.
+
+Quicksilver II hardware:
+- Main CPU: Intel Celeron 333/366MHz
+- Motherboard: Intel SE440BX-2
+- RAM: 64MB PC100-222-620 non-ecc
```

Comment:
I realise this was there before, but it doesn't seem quite right.  The wait state spec should have four digits for PC100, shouldn't it?  Like "2222" or 3111".

Reply:
You could be theoretically correct about the wait state but the part number itself is correct: https://www.digchip.com/datasheets/parts/datasheet/000/PC100-222-620-pdf.php